### ISSUE TITLE
Comment 9

### ISSUE THREAD

**Issue by ak2253** [NeverFlight/CS435-Project2-Part2#5]
When the astar method finds a path, it breaks from the while loop but omits the destnode from the arraylist so add
`ret.add(curr);`
in between lines 231 and 232 in:
https://github.com/NeverFlight/CS435-Project2-Part2/blob/d2214e6197e00763e03a18aacb510f210ff396cb/src/com/cs435/part2/Main.java#L229-L233

**Comment by NeverFlight** [comment]
Yeah, I was not sure that if we need to add the last node before. But I will do that to make the return list complete. Thank you!
https://github.com/NeverFlight/CS435-Project2-Part2/commit/41835259a7b23324f577c734f12a82c42b501493
